configuration.yaml

```diff
@@ -60,11 +44,6 @@ variables:
           particularly the client mesh master
       type: int
       default_value: 300
-    XRT_LOCAL_WORKER:
-      description:
-        - Local service address for XRT local worker, e.g. localhost:8000.
-      type: string
-      default_value: ""
     XRT_SHARD_WORLD_SIZE:
```

Comment:
Why is this flag around?

Reply:
will delete later as a task in clean XRT in Migrate to pull XLA from OpenXLA

Reply:
will delete in Will's PR to delete XRT: https://github.com/pytorch/xla/pull/5240